tst/com/amazon/ion/benchmark/OptionsTest.java

```diff
@@ -718,6 +758,51 @@ public void readBothTextAndIonUsingBothDomAndReader() throws Exception {
         assertTrue(expectedCombinations.isEmpty());
     }
 
+    @Test
+    public void readBothTextAndIonUsingIonElementDom() throws Exception {
```

Comment:
```suggestion
    public void readBothTextAndBinaryUsingIonElementDom() throws Exception {
```
Note: I see that I initially created this naming pattern with the method above. Oops!